pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/design/xml/CyclomaticComplexity.xml

```diff
@@ -263,7 +263,7 @@ trigger CaseAssignLevel on CaseAssignLevel__c (after delete, after insert, after
         <rule-property name="methodReportLevel">1</rule-property>
         <expected-problems>1</expected-problems>
         <expected-messages>
-            <message>The trigger 'CaseAssignLevel' has a cyclomatic complexity of 9.</message>
+            <message>The trigger 'CaseAssignLevel' has a cyclomatic complexity of 12.</message>
```

Comment:
This actually undoes what I've done in https://github.com/pmd/pmd/commit/581b6db3d26627e2a7955352ef3cbfdf367d6e22 during the update of the apex-parser/summit-ast